src/main/java/com/suse/saltstack/netapi/event/EventStream.java

```diff
@@ -64,13 +72,11 @@ public EventStream() {
      *
      * @param config Contains the necessary details such as EndPoint URL and
      * authentication token required to create the WebSocket.
+     * @throws SaltStackException in case of an error during stream initialization
      */
-    public EventStream(ClientConfig config) {
-        try {
-            initializeStream(config);
-        } catch (SaltStackException e) {
-            e.printStackTrace();
-        }
+    public EventStream(ClientConfig config) throws SaltStackException {
+        maxMessageLength = config.get(ClientConfig.WEBSOCKET_MAX_MESSAGE_LENGTH);
```

Comment:
OK, this alters the very idea of disabling the check and _implicitly_ set to the integer maximum and now it makes it "Either less than maximum but more than zero, or maximum". This makes perfect sense.